packages/core/src/components/table/table-body-row/table-body-row.tsx

```diff
@@ -98,12 +129,16 @@ export class TdsTableBodyRow {
   render() {
     return (
       <Host
+        tabindex={this.clickable ? '0' : null}
         class={{
           'tds-table__row': true,
           'tds-table__row--selected': this.selected,
           'tds-table__compact': this.compactDesign,
           'tds-table--divider': this.verticalDividers,
+          'tds-table__row--clickable': this.clickable,
         }}
+        onClick={() => this.handleRowClick()}
```

Comment:
```suggestion
        onClick={(e) => this.handleRowClick(e)}
```

Reply:
and them pass the event to the emitter

Reply:
Thank you for the feedback @mJarsater  👍